### PR TITLE
pygithub LGPL curation

### PR DESCRIPTION
**Pull request by clearlydefinedbot**
**Type:** Incorrect

**Summary:**
pygithub LGPL curation

**Details:**
pygithub is under LGPL-3.0-or-later under this version and others 

**Resolution:**
https://github.com/PyGithub/PyGithub/blob/v1.44.1/setup.py
https://github.com/PyGithub/PyGithub/blob/v1.44.1/COPYING.LESSER
https://pypi.org/project/PyGithub/1.44.1/

**Affected definitions**:
- [pygithub 1.44.1](https://clearlydefined.io/definitions/pypi/pypi/-/pygithub/1.44.1/1.44.1)

### CODE REVIEW

File: curations/pypi/pypi/-/pygithub.yaml [hunk 1]
```diff
@@ -6,3 +6,6 @@ revisions:
   1.43.7:
     licensed:
       declared: LGPL-3.0-or-later
+  1.44.1:
+    licensed:
+      declared: LGPL-3.0-or-later
```
Comment (dangoor) [inline curations/pypi/pypi/-/pygithub.yaml:11]:
```suggestion
      declared: LGPL-3.0-or-later
    1.51
      licensed:
        declared: LGPL-3.0-or-later
    1.53
      licensed:
        declared: LGPL-3.0-or-later
    1.56
      licensed:
        declared: LGPL-3.0-or-later
```

File: curations/pypi/pypi/-/pygithub.yaml [hunk 2]
```diff
@@ -6,3 +6,6 @@ revisions:
   1.43.7:
```
Comment (dangoor) [inline curations/pypi/pypi/-/pygithub.yaml:6]:
```suggestion
  1.1:
      licensed:
        declared: LGPL-3.0-or-later  
  1.43.7:
```

**Comment by dangoor** [comment]
I added suggestions for other versions we're using. Confirmed that even PyGitHub 1.1 is LGPL: https://pypi.org/project/PyGithub/1.1/

**Review by capfei — APPROVED** [review]

**Review by capfei — APPROVED** [review]

**Review by capfei — APPROVED** [review]

**Comment by capfei** [comment]
Finally made the validator happy! 👍

**Comment by dangoor** [comment]
Oops! Sorry for forgetting the `:`s. Editing-via-suggestion was an experiment... Editing was not the easiest there.

**Pull request merged**